packages/forms/frontend/sirius-components-forms/src/propertysections/PropertySectionLabel.tsx

```diff
@@ -44,8 +34,22 @@ const usePropertySectionLabelStyles = makeStyles((theme) => ({
 export const PropertySectionLabel = ({ editingContextId, formId, widget }: PropertySectionLabelProps) => {
   const classes = usePropertySectionLabelStyles();
 
+  const propertySectionPanelDecorators: ComponentExtension<PropertySectionLabelDecoratorProps>[] = useComponents(
+    propertySectionLabelDecoratorContributionsExtensionPoint
+  );
   if (!widget.label && !widget.hasHelpText) {
-    return null;
+    return (
+      <div>
+        {propertySectionPanelDecorators.map(({ Component: PropertySectionPanelDecorator }, index) => (
+          <PropertySectionPanelDecorator
+            editingContextId={editingContextId}
+            formId={formId}
+            widget={widget}
+            key={index}
+          />
+        ))}
```

Comment:
In case there are no label and multiple decorators I think that some layout will be necessary, I'll look into it